voice/bxml/callbacks/conferenceCreated.md

```diff
@@ -1,19 +1,32 @@
 {% method %}
 ## Conference Created Event
-The Conference Created event is fired whenever a new [conference](../verbs/conference.md) that specified a callbackUrl is created.
+The Conference Created event is fired whenever a new [conference](../verbs/conference.md) that specified a `callbackUrl` is created.
+
+The response may be either empty or a BXML. Only `<PlayAudio>` and `<SpeakSentence>` are supported, which will be heard by members that enter the conference while still playing.
```

Comment:
```suggestion
The response may be either empty or a BXML. Only `<PlayAudio>` and `<SpeakSentence>` verbs are supported, which will be heard by all members of the conference.
```

Reply:
I wrote it that way because this is when the conference just started. I thought would be relevant to say that members who enter afterwards will hear it if it's still playing.